examples/string.golden

```diff
@@ -2,3 +2,40 @@
 "hello world" : String
 (true) : Bool
 (false) : Bool
+"cde" : (Maybe String)
```

Comment:
Shouldn't this have a `just` if it's a `Maybe String`?

Reply:
Absolutely! Good catch!